_ml-commons-plugin/extensibility/connectors.md

```diff
@@ -1,135 +1,44 @@
-
-As of OpenSearch 2.9, connectors have been tested for the following ML tools, though it is possible to create connectors for other tools not listed here:
-
-- [Amazon SageMaker](https://aws.amazon.com/sagemaker/) allows you to host and manage the lifecycle of text-embedding models, powering semantic search queries in OpenSearch. When connected, Amazon SageMaker hosts your models and OpenSearch is used to query inferences. This benefits Amazon SageMaker users who value its functionality, such as model monitoring, serverless hosting, and workflow automation for continuous training and deployment.
-- [ChatGPT](https://openai.com/blog/chatgpt) enables you to run OpenSearch queries while invoking the ChatGPT API, helping you build on OpenSearch faster and improving the data retrieval speed for OpenSearch search functionality.
-
-Additional connectors will be added to this page as they are tested and verified. 
-
-
-## Prerequisites
-
-If you are an admin deploying an ML connector, make sure that the target model of the connector has already been deployed on your chosen platform. Furthermore, make sure that you have permissions to send and receive data to the third-party API for your connector. 
-
-When access control is enabled on your third-party platform, you can enter your security settings using the `authorization` or `credential` settings inside the connector API.
-
-### Adding trusted endpoints
-
-To configure connectors in OpenSearch, add the trusted endpoints to your cluster settings using the `plugins.ml_commons.trusted_connector_endpoints_regex` setting, which supports Java regex expressions, as shown in the following example:
-
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.trusted_connector_endpoints_regex": [
-            "^https://runtime\\.sagemaker\\..*\\.amazonaws\\.com/.*$",
-            "^https://api\\.openai\\.com/.*$",
-            "^https://api\\.cohere\\.ai/.*$",
-            "^https://bedrock\\..*\\.amazonaws.com/.*$"
-        ]
-    }
-}
-```
-{% include copy-curl.html %}
-
-### Enabling ML nodes
+You can provision connectors in two ways:
 
-Most connectors require the use of dedicated ML nodes. To make sure you have ML nodes enabled, update the following cluster settings:
+1. An [external connector](#external-connector), saved in a connector index, which can be reused and shared with multiple remote models but requires access to both the model, the connector inside of OpenSearch, and the third party being accessed by the connector, such as OpenAI or SageMaker.
 
-```json
-PUT /_cluster/settings
-{
-    "persistent": {
-        "plugins.ml_commons.only_run_on_ml_node": true,
-    }
-}
-```
-{% include copy-curl.html %}
+2. An [local connector](#local-connector), saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
```

Comment:
```suggestion
2. A [local connector](#local-connector), saved in the model index, can only be used with one remote model. Unlike a standalone connector, users only need access to the model itself to access an internal connector because the connection is established inside the model.
```